ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/end_to_end_example/func.py

```diff
@@ -8,6 +8,8 @@
 object_storage_client = oci.object_storage.ObjectStorageClient(config={}, signer=signer)
 data_flow_client = oci.data_flow.DataFlowClient(config={}, signer=signer)
 
+SUPPORTED_EVENT_SOURCES = {"ObjectStorage"}
```

Comment:
What are we looking for from this?

Reply:
Thanks for checking on this. It was an unintentional include. Removed in the latest commit.

Reply:
I am OK with this PR already. Is there anything else?